package.json

```diff
@@ -98,7 +98,8 @@
   "dependencies": {
     "dotenv": "16.0.3",
     "invariant": "2.2.4",
-    "pm2": "^5.2.2"
+    "pm2": "^5.2.2",
+    "stripe": "^12.1.1"
```

Comment:
Why is stripe being installed in this PR?

Reply:
hello @naomi-lgbt 
I have not installed stripe in my repository and It already existed when I cloned the repository.

Reply:
Please could you revert the changes to the package.json and pnpm-lock.yaml. Neither file needs to have changed for this PR.